src/hooks.js

```diff
@@ -0,0 +1,86 @@
+import {useEffect, useLayoutEffect} from 'react'
+import debounce from 'lodash.debounce'
+import countryIso from 'country-iso'
+import countryData from 'country-data'
+
+import drawChart from './chart'
+import getCountryData from './consumer'
+import geolocationStates from './geolocationStates'
+
+const google = window.google
+
+const getCountrySlug = coords => {
+  const code = countryIso.get(coords.latitude, coords.longitude)
+  return countryData.countries[code].name
+}
+
+const setUpGeolocation = async ({coords, setGeolocationState, setSelectedCountries, setData}) => {
+  const slug = getCountrySlug(coords).toLowerCase()
+  console.log(slug)
```

Comment:
Whoops

Reply:
Yeah maybe let's get rid of that :slightly_smiling_face: